en/contributing/build.md

```diff
@@ -44,17 +44,21 @@ To build the *DroneCore* C++ Library on Linux (or macOS after installing the [pr
      ```
 
 1. (Optionally) "Install" DroneCore [as described below](#install-artifacts). This is required in order to build DroneCore applications, but not to run DroneCore test code.
-   
+
 1. (Optionally) Build the API Reference documentation by calling:
    ```sh
    make docs
    ```
 
 ### Windows
 
-To build the library you'll need to install *Visual Studio Community Edition*. You will also need to download the [curl](https://curl.haxx.se/) source, extract and build it, and make the directory of the header files available when you build DroneCore.
+To build the library in Windows, you need:
+
+- [Build Tools for Visual Studio 2017](https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017): Download and install them if you don't already have it.
+- [curl](https://curl.haxx.se/): Download the source, extract and build it, and make the directory of the header files available when you build DroneCore.
```

Comment:
I'm not what this really means: "and make the directory of the header files available when you build DroneCore."

But anyway, can you please reorder this so that curl is afterthe cmake bullet point.

Reply:
It's what is described below, let me change that.